### ISSUE TITLE
D2GS crash

### ISSUE THREAD

**Issue by publicg4** [tesseract2048/d2gs#16]
i have a lan server for median xl 2012 .005 using d2gs and pvpgn ODBC compiled through magic builder. It works fine no crashes for a year. my friends got tired of of that version and I decided to update to median ultimative XVc D2GS crashes or refuses to run

**Comment by diablofanclub** [comment]
i think MXL XVC is only client patch , not server patch

**Comment by kryshtof** [comment]
Since v7 there's special server side patch, client patch doesn't work with d2gs.
Last MXL ultimative version that worked on server was v6f.

**Comment by publicg4** [comment]
is the server side patch of ,median xl ultimative XVc publicly available? where can i get it?

**Comment by kryshtof** [comment]
nope, it's not available - use some [older version](http://forum.median-xl.com/viewtopic.php?f=44&t=1325)

**Comment by diablofanclub** [comment]
https://github.com/lolet/D2Ex2
i like it and d2gs cant load too

**Comment by LiamKarlMitchell** [comment]
It's 2019, any server side patch available for Median XL now?
